yt/visualization/volume_rendering/input_events.py

```diff
@@ -342,6 +367,19 @@ def debug_buffer(event_coll, event):
     print(buffer.min(), buffer.max())
 
 
+@register_event("screenshot")
+def screenshot(event_coll, event):
+    """Save a copy of the displayed image"""
+    buffer = event_coll.scene.image[:]
+    n = 0
+    base = "yt_vr_%04i.png"
+    while os.path.exists(base % n):
+        n += 1
+    fn = base % n
+    print(f"Writing {fn}")
```

Comment:
Use logger functions? Same at line 367.

Reply:
I'm holding off on this for now because I want to check the way stdin/stdout and loggers get pulled versus print functions in the ipython interactive loop